gcloudutils/tags.go

```diff
@@ -0,0 +1,77 @@
+package gcloudutils
```

Comment:
This file is missing yuval's latest changes involving installation ID

Reply:
Cool, there are a bunch of changes missing and a bunch of TODOs -- I'll wait a bit and then update this.